tests/unit/Bridge/Symfony/Console/RunCommandJobLauncherTest.php

```diff
@@ -20,7 +23,12 @@ public function testLaunch(): void
         $commandRunner->runAsync('yokai:batch:run', 'test.log', $arguments)
             ->shouldBeCalledTimes(1);
 
-        $launcher = new RunCommandJobLauncher(new JobExecutionFactory(), $commandRunner->reveal(), 'test.log');
+        /** @var JobExecutionStorageInterface|ObjectProphecy $storage */
+        $storage = $this->prophesize(JobExecutionStorageInterface::class);
+        $storage->store(Argument::type(JobExecution::class))
```

Comment:
You should also asset some values of that argument. ID, status and configuration for instance